src/Glpi/Form/QuestionType/QuestionTypeItemDropdown.php

```diff
@@ -273,11 +274,20 @@ public function getDropdownRestrictionParams(?Question $question): array
                         ),
                     };
 
-                    $params[$itemtype::getTableField($key)] = 1;
+                    $type_params['AND']['OR'][] = [
+                        $itemtype::getTableField($key) => 1,
+                    ];
                 }
+
+                $params[] = $type_params;
```

Comment:
It does not change the result, but I find it easier to read and it will simplify the SQL criteria array.

```suggestion
                    $type_params[$itemtype::getTableField($key)] = 1;
                }

                $params[] = ['OR' => $type_params];
```